docs/whats_new.md

```
@@ -1302,7 +1302,7 @@ INFO Total process count:       1
 
 </details>
 
-## 📃 Log
+##  Log
```

markdownlint-cli2 (0.23.2)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)